src/aoc2020/day1.clj

```diff
@@ -0,0 +1,63 @@
+(ns aoc2020.day1
+  (:require [clojure.java.io :as io]
+            [clojure.string :as str]))
+
+(defn file->seq
+  [filename]
+  (-> filename
+      (io/resource)
+      (slurp)
+      (str/split-lines)))
+
+
+(defn sum-2020->mutiply
```

Comment:
This function is complecting multiple ideas. i.e., decoding/parsing, complement building, finding a complement...

You can tell b/c the REPL has a single function call... it's all jammed into one place.

Decompose to the relevant parts.